spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RNN.scala

```diff
@@ -28,13 +27,13 @@ class RnnCell[T : ClassTag] (
   inputSize: Int = 4,
   hiddenSize: Int = 3,
   activation: TensorModule[T],
-  private var initMethod: InitializationMethod = Default)
+  private var initMethod: InitializationMethod = Xavier)
```

Comment:
I do not know what do you mean by "wrong". But this is only a change of name not a change of behavior. You should checkout the original code. In the reset method of RNN, if it matches the "Default" initMethod, it does a Xavier initialization. In this sense, Xavier is the default initialization method of RNN.